src/main/java/org/folio/innreach/config/KafkaListenerConfiguration.java

```diff
@@ -121,9 +121,13 @@ public DefaultErrorHandler errorHandler() {
     DefaultErrorHandler errorHandler = new DefaultErrorHandler((consumerRecord, exception) -> {
       log.info("inside errorHandler for Ongoing contribution");
       // logic to execute when all the retry attempts are exhausted
-      executionService.runTenantScoped(getContributionJobContext().getTenantId(),
+      try {
+        executionService.runTenantScoped(getContributionJobContext().getTenantId(),
           () -> contributionJobRunner.completeContribution(getContributionJobContext()));
-      endContributionJobContext();
+        endContributionJobContext();
+      } catch (Exception ex) {
+        log.warn("Exception while processing error handler {} ", ex.getMessage());
+      }
```

Comment:
I see we want a try catch here just so that we don't trigger retry at all for ongoing. Do I have this right?

Reply:
No steve, this lambda function will gets invoked only when all the retries are exhausted. We still have the retry for ongoing

Reply:
Ok thanks for clarifying